GAE/src/org/waterforpeople/mapping/app/web/rest/security/featureflag/EntryPoint.java

```diff
@@ -0,0 +1,34 @@
+package org.waterforpeople.mapping.app.web.rest.security.featureflag;
```

Comment:
FYI To differentiate these we have been trying to move as many of the newer classes to new packages under the `org.akvo` domain whenever possible.

Reply:
Plus not forgetting the licensing info in general. 👍